### PR TITLE
Adding qss

### PR DESCRIPTION
**Pull request by pluflou**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pluflou** [comment]
I agree to be a maintainer. @conda-forge/help-python, ready for review!

File: recipes/qss/bld.bat [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+  -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+  -DBUILD_SHARED_LIBS=OFF ^
+  -DUNIT_TESTS=OFF ^
+  -DBUILD_SAMPLES=OFF
+if errorlevel 1 exit 1
+
+cmake --build build --parallel --config Release
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/qss/bld.bat:14]: Is there a `make check` or `ctest` we can run here?

Comment (ocefpaf) [reply recipes/qss/bld.bat:14]: I could not find any compiled parts that require cmake upstream. I'm probably missing something. Can you clarify why you need this for Windows and not Linux? Can this package be noarch?

Comment (pluflou) [reply recipes/qss/bld.bat]: @ocefpaf this package should be noarch. I was having issues with the Windows CI build failing and I couldn't figure out why, so far the only thing that fixed it was adding a bld.bat file. I don't understand why though and I was hoping someone with more experience with windows could chime in. I went ahead and deleted the file again for you to see the Windows build failure, perhaps I missed something in the logs. Thank you!

Comment (ocefpaf) [reply recipes/qss/bld.bat]: Looks like everything it working as noarch. Merging it!

**Pull request merged**